src/app/screens/Keysend/index.tsx

```diff
@@ -107,10 +131,23 @@ function Keysend() {
           </Container>
         </>
       ) : (
-        <Container maxWidth="sm">
-          {destination && <PublisherCard title={destination} />}
+        <Container justifyBetween maxWidth="sm">
+          <ResultCard
+            isSuccess
+            message={
+              !destination
+                ? successMessage
+                : `${amount} SATS ${
```

Comment:
I think we have lowecase `sats` normally. 

at least we should have this everywhere the same and I think capitalized letters are "screaming" lowercase is fine.

Reply:
We could align this. Currently we might use `tcommon('sats')`, `SATS` and `sats`.  
To avoid peopel adding "transaltions" for "sats" we should use "sats" as a hard-coded string I think.  
@im-adithya you do align this in a follow up PR.

Reply:
Sure! Opened #1624

Reply:
Maybe is a good idea to pluralize globally the word *sats* like _1 sat_ vs. _21 sats_?

Reply:
Done here itself! https://github.com/getAlby/lightning-browser-extension/pull/1581/commits/96d1aff48ce0100e74b7b32d478e256e64945677